drivers/hmis/app/models/hmis/ce/match/candidate.rb

```diff
@@ -23,17 +23,18 @@ class Candidate < GrdaWarehouseBase
 
       scope = opportunity.candidate_pool.candidates
 
+      # TODO(#7835) - descoped for now since this is getting reworked in #7835
       # do we need to allow a referral to be re-started for the same client/opportunity?
```

Comment:
I've removed this for now since my understanding is that it's getting reworked soon anyway.

Also, I think that the answer to the question on line 27 "do we need to allow a referral to be re-started?" is **yes**, at least for mvp, because we don't currently have any way of cancelling a referral that was started by accident, aside from rejecting the referral.

Reply:
Agreed, I made a commit to just remove this commented-out code. The new approach in this pR https://github.com/open-path/Green-River/issues/7835 doesn't touch this scope.